app/views/rdb_dashboard/issues/_issue_properties.html.slim

```diff
@@ -16,3 +16,11 @@
 - if User.current.allowed_to?(:view_time_entries, @project)
   .rdb-property.rdb-property-time class=(issue.estimated_hours.nil? && issue.time_entries.empty? ? 'rdb-disabled' : '')
     = t(:rdb_property_time, estimated: issue.estimated_hours.to_f.round(2), actual: issue.time_entries.map(&:hours).reduce(&:+).to_f.round(2)).html_safe
+- if issue.attachments.any?
+  %div.rdb-property.rdb-property-attachments
```

Comment:
[slim](https://github.com/slim-template/slim) isn't using `%` to indicate tags. They aren't needed here.